import-scripts/annotation_suite/standardize_mutation_data.py

```diff
@@ -0,0 +1,1391 @@
+DEFAULT_VALIDATION_STATUS = "Unknown"
+DEFAULT_VERIFICATION_STATUS = "Unknown"
+DEFAULT_MUTATION_STATUS = "Somatic"
+
+# GLOBALS FOR VALIDATING RESULTS
+VALID_VARIANT_CLASSIFICATIONS = [
+    "Frame_Shift_Del",
+    "Frame_Shift_Ins",
+    "In_Frame_Del",
+    "In_Frame_Ins",
+    "Missense_Mutation",
+    "Nonsense_Mutation",
+    "Silent",
+    "Splice_Site",
+    "Translation_Start_Site",
+    "Nonstop_Mutation",
+    "3'UTR",
+    "3'Flank",
+    "5'UTR",
+    "5'Flank",
+    "IGR",
+    "Intron",
+    "RNA",
+    "Targeted_Region"
+]
+
+VALID_VARIANT_TYPES = ["SNP", "DNP", "TNP", "ONP", "DEL", "INS"]
+
+MUTATION_FILTER = ["3'Flank", "3'UTR", "5'Flank", "5'UTR", "IGR", "Intron", "RNA", "Silent"]
+
+MUTATION_STATUS_FILTER = ["LOH", "Wildtype"]
+
+VALID_CHROMOSOMES = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15", "16", "17", "18", "19", "20", "21", "22", "X", "Y", "M"]
+
+# CGI GLOBALS
+CGI_VARIANT_CLASS_FILTER = [
+    "INTRON",
+    "TSS-UPSTREAM",
+    "UTR5",
+    "UTR3",
+    "UTR",
+    "SPAN5",
+    "SPAN3",
+    "SPAN",
+    "SYNONYMOUS",
+    "IGR",
+    "NO-CHANGE",
+    "UPSTREAM"
+]
+
+CGI_INDEL_VARIANT_CLASSES = [
+    "INSERT",
+    "DELETE",
+    "INSERT+",
+    "DELETE+",
+    "FRAMESHIFT"
+]
+# NO DIRECT MAPPING FOR: UTR, SPAN, FRAMESHIFT - These require additional data either from other variant classes
+# if datum contains comma-delimited variant classes (i.e., NO-CHANGE,DELETE,FRAMESHIFT for one record)
+# or value of variant type
+# > If only UTR is present then use IGR as default
+# > If only SPAN is present then use IGR as default
+# > If only FRAMESHIFT is present and variant type is INS/DEL then use Frame_Shift_Ins, Frame_Shift_Del.
+#   Otherwise, if variant type is not INS/DEL then use Missense_Mutation
+CGI_VARIANT_CLASS_MAP = {
+    "INTRON":"Intron",
+    "TSS-UPSTREAM":"5'Flank",
+    "UPSTREAM":"5'Flank",
+    "UTR5":"5'UTR",
+    "UTR3":"3'UTR",
+    "SPAN5":"5'UTR",
+    "SPAN3":"3'UTR",
+    "SYNONYMOUS":"Silent",
+    "MISSTART":"Translation_Start_Site",
+    "DONOR":"Splice_Site",
+    "ACCEPTOR":"Splice_Site",
+    "DISRUPT":"Splice_Site",
+    "NO-CHANGE":"Silent",
+    "MISSENSE":"Missense_Mutation",
+    "NONSENSE":"Nonsense_Mutation",
+    "NONSTOP":"Nonstop_Mutation",
+    "DELETE":"In_Frame_Del",
+    "INSERT":"In_Frame_Ins",
+    "DELETE+":"Frame_Shift_Del",
+    "INSERT+":"Frame_Shift_Ins"
+}
+
+# KEY COLUMN NAMES
+TUMOR_SEQ_ALLELE1_COLUMNS = ["Tumor_Seq_Allele1", "TumorSeq_Allele1"]
+TUMOR_SEQ_ALLELE2_COLUMNS = ["Tumor_Seq_Allele2", "TumorSeq_Allele2"]
+MUTATED_TO_ALLELE_COLUMN = "mutated_to_allele"
+VARIANT_TYPE_COLUMNS = ["Variant_Type", "VariantType", "mut_type", "mutation_type"]
+VARIANT_CLASSIFICATION_COLUMNS = ["Variant_Classification", "class", "Transcript architecture around variant"]
+START_POSITION_COLUMNS = ["Start_Position", "Start_position", "POS", "chromosome_start"]
+END_POSITION_COLUMNS = ["End_Position", "End_position", "chromosome_end"]
```

Comment:
no it's not from vcf2maf.. this script was written to handle more than just what we expect from GENIE since it was written while I was working on TARGET/CGCI which all submitted various MAFs generated from different pipelines.